src/pages/ServiceDetails/ServiceInfo/ServiceInfoRouteRules/RouteRuleRoute.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import * as resolve from 'table-resolver';
 import {
   checkForPath,
-  DestinationWeight,
+  DestinationWeightV1Alpha1,
```

Comment:
+1
In my following PR will remove the old RouteRule object, so this rename will be temporal.